app/src/main/java/blockeq/com/stellarwallet/activities/RecoverWalletActivity.kt

```diff
@@ -75,12 +78,22 @@ class RecoverWalletActivity : BaseActivity() {
                     launchPINView(PinType.CREATE,
                             getString(R.string.please_create_a_pin),
                             recoveryString,
+                            passphrase,
                             false)
                 } else {
                     showErrorMessage()
                 }
             }
         }
+
+        passphraseButton.setOnClickListener {
```

Comment:
Once the passphrase is set, we should indicate to the user that they have applied the passphrase option. This can be done by changing the text on the button to say `Passphrase applied`.